benchmark/dgram/bind-params.js

```diff
@@ -0,0 +1,42 @@
+﻿'use strict';
+
+const common = require('../common.js');
+const dgram = require('dgram');
+
+const numberOfOperations = 1e4;
+
+const configs = {
+  n: [numberOfOperations],
```

Comment:
I think this should just use the literal inline, since it's the only place the variable is used.